app/javascript/components/pages/SponsorUs.jsx

```diff
@@ -11,12 +13,53 @@ const SponsorUs = () => (
 
         <SharedLayout>
             <PageTitleWithContainer text="Sponsor Us" />
-            <div className="-mt-8">
+            <section className="container flex flex-col mx-auto md:max-w-[50rem] lg:max-w-[60rem] sponsorship-options">
+                <h2 className="px-10 text-2xl font-bold my-2">Sponsorship tiers</h2>
+
                 <stripe-pricing-table
                     pricing-table-id="prctbl_1PSlcMBHbLqUjvnWol3sB6vA"
                     publishable-key="pk_live_51JmdS1BHbLqUjvnWRLnx9KBZtHWLWHe67N3OpsV0FERko2K4lAZ6oyEo4pIq84YIMQPUJ8K1FY1UevXjZZPsiloq00EYai7JBQ"
                 />
-            </div>
+            </section>
+
+            <section className="mt-12 lg:mt-28 bg-wnbrb-pink-light">
+                <div className="container mx-auto flex flex-col md:flex-row gap-8 xl:gap-16 py-20 px-10 lg:px-8 xl:px-12">
+                    <Card className="w-100 md:w-1/2 flex-1 flex flex-col">
+                        <h2 className="text-2xl font-bold mb-2">Hire from WNB.rb</h2>
+                        <p className="mb-8 grow">
+                            Even if you aren&apos;t a sponsor, you can hire from our community by
+                            posting your open jobs on our job board.
+                        </p>
+                        <a
+                            href="https://buy.stripe.com/28o4gW9cZ7oKbwQ7sJ"
+                            target="_blank"
+                            rel="noopener noreferrer"
+                        >
+                            <Button type="primary" className="max-w-[8rem] ">
+                                Post a job
+                            </Button>
+                        </a>
```

Comment:
@egiurleo the job page link is sending to a stripe link

Reply:
maybe we can direct to an email while we work on the job page

Reply:
We can talk about making a job page in the future though! I just want to do the lowest-lift version possible :)

Reply:
I can understand. @egiurleo  I still need to confirm we are sending from that link to stripe too, is that correct?

Reply:
Yup, that's exactly right!